app/assets/javascripts/Remus.js

```diff
@@ -0,0 +1,41 @@
+$(document).ready(function() {
+  $('[data-unit="create"]').on('submit', function(event) {
+    event.preventDefault();
+    $form = $(event.currentTarget);
+    $.ajax({
+      type: "POST",
+      url: $form.attr('action'),
+      data: $($form).serialize(),
+      dataType: 'json',
+      success: function(unit) {
+        action = '/units/'+ unit.id;
+        $newForm = $('<form>').attr({
+          action: action,
+          method: 'delete',
+          'data-unit': 'delete'
+        });
+        $unitButton = $('<input>').attr({type: 'submit', value: 'Delete Unit'});
+        $newForm.append($unitButton);
+        //Line 20 now makes seven list items. WHY SEVEN?!?!?
+        var $unitListing = $('li').addClass('unit').attr('data-unit-id', unit.id).html(unit.name);
```

Comment:
Also the delete buttons generated through AJAX are broken, not sure if that's just because there's seven with the same ID and so on or a separate problem. As far as I can tell they're the same as the ones in Catstagram that work.